QuickDeployment/V1/QuickDeploymentClientSetupV1.md

```diff
@@ -1,5 +1,10 @@
 # Quick Deployment Client Setup
 
+> :memo: **NOTE**
+>
+> This document is for **Version 1** of the Quick Deployment Environment.
+> If you're using a newer version of QDE, please refer to [[this document|QuickDeploymentClientSetup]]
```

Comment:
```suggestion
> If you're using a newer version of QDE, please refer to [[the QDE Client Setup page|QuickDeploymentClientSetup]]
```

I think Rob prefers we use the name of the page for links instead of "here" or "this page", for clarity to the end-user. Correct me if I'm wrong, @ferventcoder.

Reply:
The reason I preferred not to here is that that name will be basically the same page name that they're currently on, so it's probably more confusing.

Reply:
It's also an accessibility issue and likely the bigger reason why I don't like it. Naming the document better that you are linking better helps folks with screen readers.

Reply:
Absolutely; but I don't know that there's a better name in this instance since it's literally "the older version of this same doc"?

Reply:
please refer to "the most up to date QDE Client Setup page"?